src/module/addandremove.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable */
+let arr = [];
+arr = JSON.parse(localStorage.getItem("arr")) || [];
+const addRemove = () => {
+  for (let i = 0; i < arr.length; i += 1) {
+    const list = document.querySelector(".container");
+    const itemOfList = document.createElement("div");
+    itemOfList.classList.add("list-class");
+    list.append(itemOfList);
+    const box = document.createElement("input");
+    box.type = "checkbox";
+    itemOfList.append(box);
+    const paragraph = document.createElement("p");
+    paragraph.classList.add("para-text");
+    paragraph.innerText = arr[i].names;
+    itemOfList.append(paragraph);
+    const editImage = document.createElement("i");
+    editImage.classList.add("fa-solid");
+    editImage.classList.add("fa-user-pen");
+    itemOfList.append(editImage);
+
+    const dotsImage = document.createElement("i");
+    dotsImage.classList.add("fa-solid");
+    dotsImage.classList.add("fa-ellipsis-vertical");
+    itemOfList.append(dotsImage);
+
+    const binImage = document.createElement("i");
+    binImage.classList.add("fa-solid");
+    binImage.classList.add("fa-trash");
+    itemOfList.append(binImage);
+    binImage.style.display = "none";
+    const deletList = () => {
+      const target = dotsImage.id;
+      arr.splice(target, 1);
+
```

Comment:
- [ ] [OPTIONAL] The functionality to delete a to-do item works very well, but the user experience could be better as it is not very intuitive. A delete icon could appear in place of those three vertical dots when a user hovers over the icon or when a user clicks a to-do item so that users will instinctively know that clicking it will result in the to-do item being deleted. Please implement this change in order to improve your project 👍